src/Microsoft.PowerShell.Commands.Utility/commands/utility/Update-TypeData.cs

```diff
@@ -1278,8 +1280,7 @@ private void ValidateTypeName()
         }
 
         /// <summary>
-        /// Takes out the content from the database and writes them
-        /// out
+        /// Takes out the content from the database and writes them out.
```

Comment:
consider explicit singular form:  'and writes it out.'

Reply:
Fixed.